### DISCUSSION TITLE
Showing internal backlinks

### DISCUSSION THREAD

**Discussion by kris-brown** [quarto-dev/quarto-cli#1353] [Q&A] [answered]
Hi, I have many pages that reference each other, and something I'd like to do is to view (say, on page "A") all the other parts of the site that hyperlink to "A". I'm happy to write some code to do this, but I'm curious what would be a recommended strategy to take.

**Comment by cderv** [comment, marked as answer]
For reference, related issue about back link https://github.com/quarto-dev/quarto-cli/issues/1585
